ui/src/libs/types.tsx

```diff
@@ -25,6 +34,9 @@ export interface IAPIInfo {
     group_index: number;
     responses: string[];
     examples: IAPIExample;
+    fieldInfo: IAPIFieldInfos;
+    rulesOrder: string[];
+    tags: string;
```

Comment:
Apply to other naming in this PR too.

If is a list / collection, use plural.
Else use singular.